cmdimport.go

```diff
@@ -30,9 +30,12 @@ type Definitions struct {
 
 // Keep track of the asymmetry of import subcommand.
 // When importing, the up direction wants two parameters:
-//   terraform import res-address res-id
+//
```

Comment:
this is probably my editor doing the formatting. If you wish I can revert this change

Reply:
This should be `go fmt` (invoked by the editor); it changed in Go 1.19.
Could you try with `go fmt ./...` ?

For the difference between `gofmt` and `go fmt` (🤯 ) see https://www.youtube.com/watch?v=i8zjh-PW7bg

Reply:
> Would it be possible to add a repro test, that fails with the current terravalet and passes with this PR ?

I am not sure how I would do that. Its not a problem of terravalet. Its how terraform would interpret:
```
terraform import \
    'module.github.github_branch_protection_v3.settings["test-import-gh:master"]' multi word address
```
compared to:
```
terraform import \
    'module.github.github_branch_protection_v3.settings["test-import-gh:master"]' 'multi word address'
```

Reply:
I missed https://github.com/Pix4D/terravalet/pull/51#discussion_r1112706478